modules/ROOT/pages/organization.adoc

```diff
@@ -35,44 +31,44 @@ An Anypoint Platform user who has been granted organization administrator privil
 
 * View a hierarchical tree of all of the organizations you have permissions to view. You can view and edit properties of an organization by clicking the name.
-* Click the name of a group to view and edit information about it. What you can view and edit depends on your role.
-
+* Click the name of a group to view and edit its information. What you can view and edit depends on your role.
++
 [NOTE]
---
 Changing the name or domain name of an organization changes the deep links to any existing API Portals in your organization.
---
-
 * Add and delete business groups (if enabled).
 
 === Access the Organization page:
 
-. Sign into your Anypoint Platform account as a user with the Organization Administrators role.
-. Select the *Access Management* option from the main page, or select it from the left side bar by clicking the _menu_ button in the top left corner.
+. Sign into your Anypoint Platform account as a user with the *Organization Administrators* role.
+. Select the *Access Management* option from the main page, or click the menu button in the top left corner.
 +
 image::anypoint-platform-nav-menu.png[]
 +
-The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled). +
+The *Organization* page is displayed, where you can view organizations and business groups and add new business groups (if enabled). 
+
 You can also click *Organization* in the left menu to view the Organization page.
 
 === Client ID and Client Secret
 
-Each master organization, environment, and business group within the master organization has its own associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment that you create, and they are globally unique.
+Each master organization, environment, and business group within the master organization has its own associated unique client ID and client secret. These are used for authentication by users who are not organization administrators to access assets within an organization. The client ID and password are generated by Anypoint Platform for each environment you create, and they are globally unique.
 
 To deploy proxies or APIs to CloudHub, you must use these values to configure a customer-hosted Mule Runtime or legacy API Gateway.
 
 [NOTE]
---
-Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, you must instead use the client ID and client secret for an _environment_. See xref:environments.adoc[Environments].
---
+Organization-level client IDs and client secrets are supported only for backward compatibility. In newer Anypoint Platform accounts, you must instead use the client ID and client secret for an environment. See xref:environments.adoc[Environments].
```

Comment:
"Must" carries a strong meaning that we can't enforce. Prefer "should", here. But I believe this is out of scope for Access Management and should be documented in API Manager